### DISCUSSION TITLE
Low FPS in the live view

### DISCUSSION THREAD

**Discussion by Stupoo27** [blakeblackshear/frigate#3066] [Ask A Question] [answered]
Hello,

Can anyone help me increase my FPS in the live view section? I have looked everywhere in the config and can't see how I increase it. The FPS must be about 5fps, if the debug view its showing 5fps as well. The camera feed runs at 30fps if I view it in VLC player.

Any advice you be grateful.

When I view in the frigate card it get about 3fps. 

I'm running home assistant on the old i5 desktop PC so it's not a processor/hardware issue. I'm using the Google coral as well. Logs are all clean. 

Thanks

**Comment by NickM-27** [comment, marked as answer]
Hi, so there's a few things to consider here.

1. The general live view is capped at 10 fps. You can increase the `fps` under `detect` to 10, but this will also increase the detection fps which will increase CPU usage.
2. The debug fps is limited to 5 fps no matter what as it uses a lot of resources, there is no way to increase this.
3. The fps on the frigate card will also depend on which provider is used.

**Reply by Stupoo27** [reply]
Thanks. Maybe I'm getting the 10fps already but detect was set at 5fps and I changed it to 10fps and the live view looks the same. When you say provider for the card what options is there for that?

**Reply by NickM-27** [reply]
These are the providers, from the card documentation: https://github.com/dermotduffy/frigate-hass-card#available-live-providers

**Reply by Stupoo27** [reply]
Brilliant. Thanks guys. I changed to jsmpeg and it's perfectly smooth now
